.github/workflows/test.yml

```diff
@@ -45,26 +45,24 @@ jobs:
       # shell: bash
       run: |
         # set -e
-        mkdir $HOME/.icewm/
-        echo "ShowTaskBar = 0" > $HOME/.icewm/preferences
-        echo "TaskBarAutoHide = 1" > $HOME/.icewm/preferences
-        echo "TaskBarShowWorkspaces = 0" > $HOME/.icewm/preferences
-        echo "TaskBarShowAllWindows = 0" > $HOME/.icewm/preferences
-        echo "TaskBarShowClock = 0" > $HOME/.icewm/preferences
-        echo "TaskBarShowMailboxStatus = 0" > $HOME/.icewm/preferences
-        echo "TaskBarShowCPUStatus = 0" > $HOME/.icewm/preferences
-        echo "TaskBarShowWindowListMenu = 0" > $HOME/.icewm/preferences
+        cp -a .github/.icewm $HOME
         # xpra start :99 # Cannot get screenshots to work
         Xvfb :99 -screen 0 800x600x24 >/dev/null 2>&1 & # Need to set bit depth, otherwise get some black screenshots
         # until xset -q; do echo "Waiting for X server to start..."; sleep 1; done # We are not immediately using it anyway
         # Find out which files in data/ have been changed in the last commit
         #set -x    # Debug #379
         FILES=$(git log -1 -p data/ | grep +++ | cut -d '/' -f 2-| sed -e 's|dev/null||g')
         echo "Last changed files from the FILES variable:"
-        echo "$FILES" ; if [ -z "$FILES" ] ; then echo "Variable FILES is empty." && exit 1 ; fi
+        echo "$FILES"
+        if [ -z "$FILES" ] ; then
+          echo "Variable FILES is empty."
+          exit 1
+        fi
         # Work on these files that have been changed in the last commit
-        # Check if $FILES is not empty
-        if [ -n "$FILES" ] ; then for FILE in $FILES ; do echo "$FILE" ; bash -e code/worker.sh $(readlink -f "$FILE") 2>&1 | tee log.txt ; done ; fi
```

Comment:
I removed the `-n` check because if the program should have exited above if `-z $FILES` is true.

Reply:
Hi @andy5995, while I like this improvement, can you please restore the icewm stuff like it was? Thanks.